rclcpp/test/test_time.cpp

```diff
@@ -123,6 +123,16 @@ TEST(TestTime, operators) {
   EXPECT_ANY_THROW((void)(now > later));
   EXPECT_ANY_THROW((void)(now + later));
   EXPECT_ANY_THROW((void)(now - later));
+
+  for (auto time_source : {RCL_ROS_TIME, RCL_SYSTEM_TIME, RCL_STEADY_TIME}) {
+    rclcpp::Time time = rclcpp::Time(0, 0, time_source);
+    rclcpp::Time copy_constructor_time = time;
```

Comment:
This should be `rclcpp::Time copy_constructor_time(time)` if you want to call the copy constructor.